bin/webpack.js

```diff
@@ -318,7 +318,7 @@
 			});
 
 			if (typeof outputOptions.colors === "undefined")
-				outputOptions.colors = require("supports-color");
+				outputOptions.colors = require("supports-color").stdout;
```

Comment:
No stdout here

Reply:
Removing this breaks the tests. Did you read my [above comment](https://github.com/webpack/webpack-cli/pull/290#issuecomment-369672078)?
Do we want the tests to fail?

Reply:
I want colors to work as intended. If the tests are failing you gotta fix em 👩‍💻✌️

Reply:
Coolio. So, in that case we can use something like [ansi-parser] (https://www.npmjs.com/package/ansi-parser) to parse `stdout` inside tests. WDYT?

Reply:
Remove the stdout and lemme have a look

Reply:
Done. You're up captain. :guardsman: